libs/clip/clip/Clipboard.hx

```diff
@@ -0,0 +1,20 @@
+package clip;
+
+@:hlNative("clipboard")
+class Clipboard {
+    static function get_clipboard_text () : hl.Bytes {
+        return null;
+    }
+
+    public static function getText () {
+        return @:privateAccess String.fromUTF8(get_clipboard_text());
+    }
+    
+    static function set_clipboard_text (text : hl.Bytes) : Bool {
+        return false;
+    }
+
+    public static function setText (text : String) {
+        return @:privateAccess set_clipboard_text(text.toUtf8());
+    }
+}
```

Comment:
Formatting inconsistent with Haxe stdlib, also a `Bytes` version should be even simpler than text.

Reply:
Should the haxe-formatter be used on the haxe files or what is the usual way of making sure, the formatting is correct?